src/amo/components/SearchContextCard/index.js

```diff
@@ -25,10 +31,27 @@ export class SearchContextCardBase extends React.Component {
   render() {
     const { count, filters, i18n, loading } = this.props;
     const { query } = filters;
+    const { addonType } = filters;
 
     let searchText;
+    let addOn;
 
-    if (!loading && query) {
+    if (addonType === ADDON_TYPE_EXTENSION) {
+      addOn = i18n.ngettext('extension', 'extensions', count);
+    } else if (addonType === ADDON_TYPE_DICT) {
+      addOn = i18n.ngettext('dictionary', 'dictionaries', count);
+    } else if (addonType === ADDON_TYPE_LANG) {
+      addOn = i18n.ngettext('language pack', 'language packs', count);
+    } else if (addonType === ADDON_TYPE_THEME) {
+      addOn = i18n.ngettext('theme', 'themes', count);
+    }
+
+    if (!loading && query && addonType) {
+      searchText = i18n.sprintf(i18n.gettext(
+        '%(count)s %(addOn)s found for "%(query)s"'
+      ), { count: i18n.formatNumber(count), addOn, query }
+      );
+    } else if (!loading && query && !addonType) {
```

Comment:
I think the `!addonType` is redundant; because you test for `addonType` above and this is otherwise the same check you can just remove that check here 😄